src/Make.inc

```diff
@@ -22,16 +22,32 @@ ifneq (,$(findstring MINGW,$(OS))$(findstring MSYS,$(OS))$(findstring CYGWIN,$(O
   OS := WINNT
 endif
 
+LBT_SOVERSION_MAJOR := 4
+LBT_SOVERSION_MINOR := 1
+LBT_SOVERSION_PATCH := 0
+
 ifeq ($(OS), WINNT)
   SHLIB_EXT := dll
   EXE := .exe
   binlib := bin
+
+  LIB_FULL_VERSION := libblastrampoline-$(LBT_SOVERSION_MAJOR)-$(LBT_SOVERSION_MINOR)-$(LBT_SOVERSION_PATCH).dll
+  LIB_MAJOR_VERSION := libblastrampoline-$(LBT_SOVERSION_MAJOR).dll
+  SONAME_FLAG :=
 else ifeq ($(OS), Darwin)
   SHLIB_EXT := dylib
   binlib := lib
+
+  LIB_FULL_VERSION := libblastrampoline.$(LBT_SOVERSION_MAJOR).$(LBT_SOVERSION_MINOR).$(LBT_SOVERSION_PATCH).dylib
+  LIB_MAJOR_VERSION := libblastrampoline.$(LBT_SOVERSION_MAJOR).dylib
+  SONAME_FLAG := -Wl,-install_name=@rpath/$(LIB_FULL_VERSION)
```

Comment:
Shouldn't the soname use only the major version, when you actually break the API? Otherwise each version will break the ABI

Reply:
Also, this variable isn't used anywhere, so the soname isn't actually set